charts/umbrella/README.md

```diff
@@ -330,7 +338,28 @@ TBD.
 
 #### Get to know the Portal
 
-Perform first login and send out an invitation to a company to join the network (SMTP account required to be configured in custom values.yaml file).
+Perform first login and send out an invitation to a company to join the network (SMTP account required to be configured in custom values.yaml file). Since the onboarding process requires the clearinghouse to work properly, but clearinghouse currently isn't available as a FOSS application you can skip the step with the following SQL Script which must be executed against the portal database.
+
+```sql
+WITH applications AS (
+    SELECT distinct ca.id as Id, ca.checklist_process_id as ChecklistId
+    FROM portal.company_applications as ca
+             JOIN portal.application_checklist as ac ON ca.id = ac.application_id
+    WHERE 
+      ca.application_status_id = 7 
+    AND ac.application_checklist_entry_type_id = 6
+    AND (ac.application_checklist_entry_status_id = 4 OR ac.application_checklist_entry_status_id = 1)
```

Comment:
Currently, the clearinghouseTokenAddress default value is set to "http://keycloak.tx.test/realms/example/protocol/openid-connect/token". When the process worker makes a request to Keycloak with this default address, the request fails. Despite the failure, the status_id remains set to 3.

We could change `ac.application_checklist_entry_status_id = 4` condition to `ac.application_checklist_entry_status_id = 3`, change the default clearinghouseTokenAddress to centralidp to make it fail or explain it.

I'm not sure which could be the best approach

Reply:
![Screenshot 2024-08-07 at 17-38-03 Mondragon-X Portal](https://github.com/user-attachments/assets/f3fa6e76-dd90-4131-b836-69512ed10952)